### ISSUE TITLE
0.9.1 Compatibility

### ISSUE THREAD

**Issue by bkonkle** [hdgarrood/purescript-sequences#22]
0.9.1 has a problem with FingerTree. 😅

``` haskell
[1/1 ErrorParsingModule] bower_components/purescript-sequences/src/Data/FingerTree.purs:96:10

  96    foldr (-<) z (Node2 _ a b)   = a -< (b -< z)


  Unable to parse module:
  unexpected -<
  expecting binder
```

Is it incorrectly interpreting this as a record update and then rejecting it because of the deprecated syntax?

Thanks for any help you can provide!

**Comment by hdgarrood** [comment]
Not quite, the problem is that operators must be declared as aliases, but here I'm trying to bind one by calling it ‘-<‘ without using an alias. I'd fix it just by calling it `f` instead.

**Comment by bkonkle** [comment]
Oh, that makes total sense! I didn't realize that by using them in pattern matching it was actually introducing them to the local scope. I'll send a PR your way this (US) morning. 👍

**Comment by bkonkle** [comment]
Okay, looks like the following dependencies need to be updated as well:

* purescript-benchotron
* purescript-node-fs
* purescript-quickcheck-laws

Here's the build output so far:

```purescript
brandon at Ryujin in ~/code/purescript-sequences branch master[M?]
± npm run build

> @ build /Users/brandon/code/purescript-sequences
> pulp build

* Building project in /Users/brandon/code/purescript-sequences
[1/8 ErrorParsingModule] bower_components/purescript-benchotron/src/Benchotron/StdIO.purs:9:8

  9  import qualified Data.String as S


  Unable to parse module:
  unexpected "qualified"
  expecting qualifier or module name

[2/8 ErrorParsingModule] bower_components/purescript-benchotron/src/Benchotron/UI/Console.purs:9:8

  9  import qualified Data.Array as A


  Unable to parse module:
  unexpected "qualified"
  expecting qualifier or module name

[3/8 ErrorParsingModule] bower_components/purescript-benchotron/src/Benchotron/Utils.purs:16:1

  16  (>>) :: forall m a b. (Bind m) => m a -> m b -> m b


  Unable to parse module:
  unexpected (
  expecting declaration or end of input

[4/8 ErrorParsingModule] bower_components/purescript-node-fs/src/Node/FS/Perms.purs:40:16

  40    eq (Perm { r = r1, w = w1, x = x1 }) (Perm { r = r2, w = w2, x = x2 }) =


  Unable to parse module:
  unexpected =
  expecting :, , or }

[5/8 ErrorParsingModule] bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws/Control/Applicative.purs:44:48

  44    interchange y u = (u <*> pure y) == (pure ($ y) <*> u)


  Unable to parse module:
  unexpected "y"
  expecting )

[6/8 ErrorParsingModule] bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws/Control/Extend.purs:24:32

  24    associativity f g x = ((f <<=) <<< (g <<=) $ x) == (f <<< (g <<=) <<= x)


  Unable to parse module:
  unexpected )
  expecting expression

[7/8 ErrorParsingModule] bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws/Data/Functor.purs:30:54

  30    composition f g x = ((<$>) (f <<< g) x) == (((f <$>) <<< (g <$>)) x)


  Unable to parse module:
  unexpected )
  expecting expression

[8/8 ErrorParsingModule] src/Data/FingerTree.purs:96:10

  96    foldr (-<) z (Node2 _ a b)   = a -< (b -< z)


  Unable to parse module:
  unexpected -<
  expecting binder

           Src   Lib   All
Warnings   0     0     0
Errors     1     7     8
```

**Comment by paulyoung** [comment]
I think the latest on this is:
- `purescript-benchotron`
  - `purescript-datetime` added support for `0.9.1` in `v1.0.0`
  - `purescript-node-fs` 0.9.1 support is waiting on https://github.com/purescript-node/purescript-node-fs/pull/30
- `purescript-node-fs` 0.9.1 support is waiting on https://github.com/purescript-node/purescript-node-fs/pull/30
- `purescript-quickcheck-laws` added support for `0.9.1` in `v1.0.0`

**Comment by kika** [comment]
`purescript-node-fs` is good to go, `1.0.0` was released today.

**Comment by bkonkle** [comment]
Looks like `purescript-benchotron` is blocked by `purescript-node-readline`.

**Comment by bkonkle** [comment]
Currently still blocked by `purescript-benchotron`.

**Comment by hdgarrood** [comment]
Fixed in #26 (kind of, we skipped over 0.9.x in the end).
